requirements/base.in

```diff
@@ -14,5 +14,5 @@ django_extensions                       # Required to use TimeStampedModel
 django-simple-history                   # History for Django models
 django-object-actions                   # Object actions in Django admin
 edx-rest-api-client                     # For accessing the Enrollment API (and possibly other edX APIs)
-django-config-models                    # Provides ConfigurationModel abstract base class
+-e git://github.com/snowcrshd/django-config-models.git@lucas/OC-2387/django-1.10-support#egg=django-config-models
```

Comment:
@brittneyexline that's just for testing while DCM is not published.
I'll update the commit when this is good.